docs/doc_vars_reference.md

```diff
@@ -0,0 +1,223 @@
+---
+id: document_variables_reference
+title: Label Variables Quick Reference
+sidebar_label: Label Variables
+slug: /label_variables
+---
+
+Lables and variables you can use in your PDFs and DOCXs to make the most of the [AssemblyLine Weaver interview code generator](https://github.com/suffolkLITLab/docassemble-assemblylinewizard). It's more work now, but it should save you a bunch of work in the future.
+
+:::info
+This is a subset of the variables in the AssemblyLine library. These are just the words that the [ALWeaver](https://github.com/suffolkLITLab/docassemble-assemblylinewizard) knows. They help you generate your starting code.
+:::
+
+## Prefixes: [Reserved words](#reserved_words) for objects
```

Comment:
I don't think that `reserved_words` (here and a few other lines later on) actually links to anything yet, is that just missing?

Reply:
It should be `reserved-words`, I fixed in ece85eb